### PR TITLE
configure workflow service

### PR DESCRIPTION
**Pull request by peetucket**

### CODE REVIEW

File: config/boot.rb [hunk 1]
```diff
@@ -33,6 +33,8 @@
 puts "Loading config from #{env_file}"
 require env_file
 
+Dor::WorkflowService.configure(Dor::Config::workflow.url)
```
Comment (jcoyne) [inline config/boot.rb]: Should this be in lyber-core?

Comment (peetucket) [reply config/boot.rb]: like this? https://github.com/sul-dlss/lyber-core/pull/35

Comment (peetucket) [reply config/boot.rb]: i guess we would presume that any consumer of the lyber-core gem can reasonably be expected to have a configuration that includes that URL

**Comment by mjgiarlo** [comment]
Moved to lyber-core.

**Pull request closed without merging**